src/main/java/ro/developmentfactory/Cynema/MainCynema.java

```diff
@@ -0,0 +1,44 @@
+package ro.developmentfactory.Cynema;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class MainCynema {
+    public static void main(String[] args) {
+        ExecutorService executorService = Executors.newFixedThreadPool(3);
+
+        AtomicInteger atomicInteger = new AtomicInteger(100);
+
+        Cynema cynema = new Cynema(atomicInteger);
+
+        Runnable task1 = () -> {
+            try {
+                System.out.println(cynema.RezervaLocuriDisponibile(10));
+            } catch (Exception e) {
+                System.out.println(e.getMessage());
+            }
+        };
+        Runnable task2 = () -> {
+            try {
+                System.out.println(cynema.RezervaLocuriDisponibile(50));
+            } catch (Exception e) {
+                System.out.println(e.getMessage());
+            }
+        };
+
+        Runnable task3 = () -> {
+            try {
+                System.out.println(cynema.RezervaLocuriDisponibile(30));
+            } catch (Exception e) {
+                System.out.println(e.getMessage());
+            }
+        };
+
+        executorService.submit(task1);
+        executorService.submit(task2);
+        executorService.submit(task3);
```

Comment:
1. you are only doing pessimistic locking calls here. please after you refactor to 2 classes: with pessimistic and optimistic locking, create both objects and do reservations to both
2. you need to process the response from the cinema, not just print the string
3. in place of creating a runnable per reservation, you can create an array of integers like `int[] reservations = {1, 2, 3, 4, 5, 10};` and then
```
int[] reservations = {1, 2, 3, 4, 5, 10};
      Arrays.stream(reservations).forEach(seatsToReserve -> {
          Runnable task = () -> {
              // handle cinema.doReservation(seatsToReserve)
          };
          executorService.submit(task);
      });
```
to avoid copy-paste. then you can even extract `// handle cinema.doReservation(seatsToReserve)` to a separate private method to make it more readable